src/Listeners/MessengerEventsSubscriber.php

```diff
@@ -75,4 +77,9 @@ public function onWorkerMessageHandled(WorkerMessageHandledEvent $event)
 
         $this->inspector->flush();
     }
+
+    public function onWorkerMessageReceived(WorkerMessageReceivedEvent $event)
```

Comment:
I had to return this code, as without it the command `php bin/console messenger:consume async -vvv` leads to exception:

```
In MessengerEventsSubscriber.php line 75:
                                                  
  [Error]                                         
  Call to a member function addContext() on null 
```

Reply:
put this code at the beginning of the method:

```php
if (!$this->inspector->hasTransaction()) {
    return;
}
```

Reply:
Do you want to remove `onWorkerMessageReceived` and put the code at the beginning of `onWorkerMessageHandled`?

Reply:
Yes, it is.